bucket/baidupcs-go.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "3.8.4",
+    "description": "Terminal utility for Baidu Network Disk",
+    "homepage": "https://github.com/qjfoidnh/BaiduPCS-Go",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/qjfoidnh/BaiduPCS-Go/releases/download/v3.8.4/BaiduPCS-Go-v3.8.4-windows-x64.zip",
+            "hash": "a9fdb978315b6e9c092f21d01393a7f826f8a9db9d7ea6d76e3af15c8d3d9277",
+            "extract_dir": "BaiduPCS-Go-v3.8.4-windows-x64"
+        },
+        "32bit": {
+            "url": "https://github.com/qjfoidnh/BaiduPCS-Go/releases/download/v3.8.4/BaiduPCS-Go-v3.8.4-windows-x86.zip",
+            "hash": "5d54a249d9f9d66e1caac8342b49a6474b2ac0c67bcd095848551a04fc612a20",
+            "extract_dir": "BaiduPCS-Go-v3.8.4-windows-x86"
+        }
+    },
+    "pre_install": "if (!(Test-Path \"$persist_dir\\pcs_config.json\")) { New-Item \"$dir\\pcs_config.json\" | Out-Null }",
+    "bin": "BaiduPCS-Go.exe",
+    "persist": [
+        "pcs_command_history.txt",
```

Comment:
You need to persist a few more directories

Look at https://github.com/hoilc/scoop-lemon/blob/master/bucket/baidupcs-go.json

Reply:
No, BaiduPCS-Go won't create `Downloads` directory. And I store the configure files alongside the exe, so we don't need to persist `config` either.

Reply:
@AlexaraWu but when it updates to the next version, would these persist files remain?

Reply:
@AlexaraWu please test and report

Reply:
It does remain, and seems not need to persist command history file.

Reply:
> It does remain

The config folder remains?

Reply:
This manifest do not create config folder, it create conig file `pcs_config.json` alongside with the exe. Tested works well.

Reply:
@AlexaraWu what about the config folder?

Why does it not need to be persisted if you persist the json file?

Reply:
The program search for configuration file in install directory firstly, and then in directory set by `BAIDUPCS_GO_CONFIG_DIR`.